com.unity.cinemachine/Tests/Runtime/CamerasBlendingTests.cs

```diff
@@ -25,165 +26,237 @@ public override void SetUp()
             // Blending
             m_Brain.m_DefaultBlend = new CinemachineBlendDefinition(
                 CinemachineBlendDefinition.Style.Linear,
-                BlendingTime);
-
-            var followObject = CreateGameObject("Follow Object");
+                k_BlendingTime);
 
             // Source vcam
-            var sourceVCam = CreateGameObject("Source CM Vcam", typeof(CmCamera)).GetComponent<CmCamera>();
-            sourceVCam.Priority = 2;
-            sourceVCam.Follow = followObject.transform;
+            m_Source = CreateGameObject("A", typeof(CinemachineVirtualCamera)).GetComponent<CinemachineVirtualCamera>();
```

Comment:
Should be conditional-compile: CinemachineVirtualCamera for pre-3.0, CmCamera otherwise